### ISSUE TITLE
Make it clearer that --verbose is an option for the help subcommand itself

### ISSUE THREAD

**Issue by scotje** [ddfreyne/cri#103]
When using the basic help command, the way the `--verbose` option is currently presented can be a bit confusing for users:

```
$ mycommand help new

<snip>

OPTIONS FOR NEW
    -d --debug                     Enable debug output.
    -h --help                      Show help for this command.
    -v --verbose                   show more detailed help
       --version                   Show version of mycommand.
```

It would be clearer if `--verbose` was listed in it's own section labelled `OPTIONS FOR HELP` or something similar:

```
$ mycommand help new

<snip>

OPTIONS FOR NEW
    -d --debug                     Enable debug output.
    -h --help                      Show help for this command.

OPTIONS FOR HELP
    -v --verbose                   show more detailed help
```

**Comment by ddfreyne** [comment]
Hey,

Which version of Cri does this happen in? 2.15.8 had a bug in it, which was rectified in 2.15.9 (see [release notes](https://github.com/ddfreyne/cri/releases/tag/2.15.9)).

I can’t reproduce the problem locally; do you have a test case?

**Comment by scotje** [comment]
Let me check again with 2.15.9 and see if I can still replicate the issue.

**Comment by scotje** [comment]
Yeah it looks like 2.15.9 resolves this. Is it expected then that the `--verbose` option is only ever listed in the `(1 hidden command omitted; show it with --verbose)` text that shows up when you hide something?

**Comment by ddfreyne** [comment]
> Is it expected then that the `--verbose` option is only ever listed in the `(1 hidden command omitted; show it with --verbose)` text that shows up when you hide something?

I don't quite understand what you mean. Can you rephrase? The `--verbose` option should always be visible in the help text.
